ObjectiveGit/GTBlob.m

```diff
@@ -40,7 +40,7 @@
 @implementation GTBlob
 
 - (NSString *)description {
-  return [NSString stringWithFormat:@"<%@: %p> size: %zi, content: %@, data = %@", NSStringFromClass([self class]), self, [self size], [self content], [self data]];
+  return [NSString stringWithFormat:@"<%@: %p> size: %%lli, content: %@, data = %@", NSStringFromClass([self class]), self, [self size], [self content], [self data]];
```

Comment:
Looks like there's an extraneous % in there.

Reply:
fixed
it was too late at night and I took a shortcut by making the change directly on github